### PR TITLE
Update index.md

### PR DESCRIPTION
**Pull request by crysacarter**
[Preview link](https://cg-f5a56614-c88e-470c-9f06-ff5fe2d4c98a.app.cloud.gov/preview/gsa/resources.data.gov/BRANCH/)

Some grammatical updates to keep with US GPO standards.

### CODE REVIEW

**Comment by adborden** [comment]
Thanks @crysacarter! Can you provide a link to the US GPO styleguide? I'm used to following the [18F content guide](https://content-guide.18f.gov/) which [explicitly does not capitalize "federal"](https://content-guide.18f.gov/specific-words-and-phrases/) unless it's part of a name. @hkdctol do you know what we've followed in the past for other data.gov sites? Best to be consistent.

**Comment by crysacarter** [comment]
Yes. Absolutely. If 18f says no, then we probably shouldn’t

**Comment by crysacarter** [comment]
https://www.govinfo.gov/content/pkg/GPO-STYLEMANUAL-2016/pdf/GPO-STYLEMANUAL-2016.pdf

**Comment by adborden** [comment]
Great, the other edits look good to me. Do you want to update the PR to undo the "federal" changes?

File: pages/index.md [hunk 1]
```diff
@@ -1,20 +1,20 @@
 ---
 layout: home
-tagline: Federal Enterprise Data Resources are tools, best practices, and schema standards to facilitate the adoption of open data practices across the federal government
+tagline: Federal Enterprise Data Resources are tools, best practices, and schema standards to facilitate the adoption of open data practices across the Federal government
```
Comment (crysacarter) [inline pages/index.md]: Federal Enterprise Data Resources are tools, best practices, and schema standards to facilitate the adoption of open data practices across the federal government.

**Pull request merged**